hack/windows/Prepare-Node.ps1

```diff
@@ -4,7 +4,7 @@
 
 .DESCRIPTION
 This script is only used for test to assist with joining a Windows node to a cluster.
-For production environment please follow the antrea windows installation guide and use kubernetes official script: https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/kubeadm/scripts/PrepareNode.ps1
+For production environment please follow the antrea windows installation guide and use kubernetes official script: https://github.com/kubernetes-sigs/sig-windows-tools/releases/download/v0.1.5/PrepareNode.ps1
```

Comment:
Ditto.

Reply:
This is the script description so I think we can keep it as it is? I have used variables for others.

Reply:
Updated.